packaging/opae/rpm/create

```diff
@@ -50,6 +50,27 @@ declare -r RPM_SPEC="opae.spec.${WHICH_RPM}"
 
 # Check for prerequisite packages.
 
+declare -ra PYTHON_PKG_DEPS=(\
+'python3-devel' \
+'python36-devel' \
+'python38-devel'\
```

Comment:
missing 3.9,3.10, and 3.11 ?
Fedora37 python3.11 version
fedora36 python3.10 version

default pyhon packages changes every Fedora release
can we use python3*-devel ?

Reply:
We can't use python3*-devel.  Per Gwen, it breaks on Fedora37.  What are the exact package names in Fedora36 and 37?

Reply:
Fedora37  cat /etc/redhat-release
Fedora release 37 (Rawhide)
python
Python 3.11.0b3 (main, Jun 24 2022, 00:00:00) [GCC 12.1.1 20220507 (Red Hat 12.1.1-1)] on linux

Fedora35  cat /etc/redhat-release
Fedora release 35 (Thirty Five)
 python
Python 3.10.4 (main, Mar 25 2022, 00:00:00) [GCC 11.2.1 20220127 (Red Hat 11.2.1-9)] on linux


Python in Fedora 36, Python 3.10,